pkg/server/pod_controller.go

```diff
@@ -486,6 +499,30 @@ func (pc *PodController) startUnit(ctx context.Context, unit api.Unit, allCreds
 	delete(pc.syncErrors, unit.Name)
 }
 
+// we want to tell tosi whether to use an overlay rootfs for containers
+// if the value returned by this function is truthy or missing we instruct
+// tosi to use the "-mount" option otherwise if value is falsy we inform
+// tosi to use the "-extractto" option
+func (pc *PodController) useImageOverlayRootfs() bool {
+	imageOverlayRootfsKey := "pod.elotl.co/image-overlay-rootfs"
+	if val, ok := pc.annotations[imageOverlayRootfsKey]; ok {
+		if val == "" {
```

Comment:
`strconv.ParseBool("")` returns an error, so I think this check isn't necessary as it could be handled in the if `err != nil` below :)

Reply:
good catch, I have updated this.